stable/mlrun/templates/db-configmap.yaml

```
@@ -49,14 +49,15 @@ data:
       --user={{- .Values.db.securityContext.runAsUser }} \
       --sql_mode="" \
       --init-file=$INIT_SCRIPT \
-      --innodb-adaptive-hash-index=0 \
-      --innodb-read-io-threads=32 \
-      --innodb-write-io-threads=32 \
-      --innodb-purge-threads=16 \
-      --innodb-buffer-pool-size=1073741824 \
-      --innodb-buffer-pool-instances=8 \
-      --innodb-page-cleaners=8 \
-      --innodb-io-capacity=10000 \
+      --max-connections={{- .Values.db.dbConfiguration.maxConnections }} \
+      --innodb-adaptive-hash-index={{- .Values.db.dbConfiguration.adaptiveHashIndex }} \
+      --innodb-read-io-threads={{- .Values.db.dbConfiguration.readIOThreads }} \
+      --innodb-write-io-threads={{- .Values.db.dbConfiguration.writeIOThreads }} \
+      --innodb-purge-threads={{- .Values.db.dbConfiguration.purgeThreads }} \
+      --innodb-buffer-pool-size={{- .Values.db.dbConfiguration.bufferPoolSize }} \
+      --innodb-buffer-pool-instances={{- .Values.db.dbConfiguration.bufferPoolInstances }} \
+      --innodb-page-cleaners={{- .Values.db.dbConfiguration.pageCleaners }} \
+      --innodb-io-capacity={{- .Values.db.dbConfiguration.IOCapacity }} \
       --innodb-flush-neighbors=0 \
```

looks like you missed one
```suggestion
      --innodb-flush-neighbors={{- .Values.db.dbConfiguration. flushNeighbors }} \
```